sysutils/dvtm/Portfile

```diff
@@ -18,23 +19,31 @@ long_description        dvtm brings the concept of tiling window management, \
                         it easy to work with multiple console based programs \
                         like vim, mutt, cmus or irssi.
 
-homepage                http://brain-dump.org/projects/dvtm/
-master_sites            ${homepage}
+homepage                https://brain-dump.org/projects/dvtm
+master_sites            ${homepage}/
 
-checksums               rmd160  36f14f5e7da72aeee7ca412d905d7d50047208d4 \
-                        sha256  7750872578e259d86c79277c19f3715c1fd3ff8d4f6c300ff8e12b270db86651
+checksums               rmd160  f4222388098891433711020b2739e5756b7c2122 \
+                        sha256  8f2015c05e2ad82f12ae4cf12b363d34f527a4bbc8c369667f239e4542e1e510 \
+                        size    40511
 
 depends_lib             port:ncurses
 
-patchfiles              patch-src-config.mk.diff
+makefile.has_destdir    yes
 
-platform darwin {
-    patchfiles-append   patch-src-Makefile.diff
-}
+build.pre_args-append   PREFIX=${prefix} \
+                        VERSION=${version}
+
+destroot.args           PREFIX=${prefix}
 
 use_configure           no
```

Comment:
The makefile 1.0 portgroup sets `use_configure no` for you.
```suggestion
```

Reply:
Good point.  Thanks @ryandesign.